helmfile.d/stacks/falco.yaml.gotmpl

```diff
@@ -1,6 +1,7 @@
 ---
 templates:
   falco:
+    disableValidationOnInstall: true
```

Comment:
Why do we need to set this to true? Is it because of the servicemonitor?

Reply:
Yes